sci-libs/libspatialindex/libspatialindex-1.9.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+MY_PN="spatialindex-src"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="General framework for developing spatial indices"
+HOMEPAGE="https://libspatialindex.org/
+	https://github.com/libspatialindex/libspatialindex"
+SRC_URI="https://github.com/libspatialindex/${PN}/releases/download/${PV}/${MY_P}.tar.bz2"
+LICENSE="MIT"
+
+KEYWORDS="~amd64 ~x86"
```

Comment:
`LICENSE` goes into the block below. Here's how this should look, in order:

```
LICENSE
SLOT
KEYWORDS
IUSE
```

Reply:
DONE